packages/@lwc/engine-dom/src/apis/hydrate-component.ts

```diff
@@ -33,42 +66,43 @@ export function hydrateComponent(
         );
     }
 
+    if (getAssociatedVMIfPresent(element)) {
+        /* eslint-disable-next-line no-console */
+        console.warn(
+            `"hydrateComponent" expects an element that is not hydrated, instead received %o.`,
```

Comment:
TIL. I didn't know that `%o` was a thing. 
Do you know what is the support for this? If the support is good, could we use it in some other places in the LWC engine?

Reply:
> Do you know what is the support for this?

I think is good: https://developer.mozilla.org/en-US/docs/Web/API/console#outputting_text_to_the_console

>If the support is good, could we use it in some other places in the LWC engine?

Sure, let's open an issue